test/services/get_stats_test.rb

```diff
@@ -6,20 +6,28 @@ class GetStatsTest < ActiveSupport::TestCase
     team = PrepareTeam.new.call(team_params)
     team.team_members.create!(slack_user_id: "user_id",
```

Comment:
temporarily it's ok, but overall try to avoid calling AR methods directly in the tests - the more we rely on service objects the safer that we rely on state which is possible to exist :)